### PR TITLE
case studies page

### PR DESCRIPTION
**Pull request by etanb**
New Case Studies page. No new components! Just content.

### CODE REVIEW

File: frontend-react/src/AppRouter.tsx [hunk 1]
```diff
@@ -249,6 +263,14 @@ export const appRoutes: RouteObject[] = [
                     },
                 ],
             },
+            {
+                path: "/product/release-notes",
+                index: true,
+                lazy: lazyRouteMarkdown("content/product/release-notes"),
```
Comment (penny-lischer) [inline frontend-react/src/AppRouter.tsx:269]: This should be content/about/release-notes

Comment (etanb) [reply frontend-react/src/AppRouter.tsx:269]: Nice catch, bad merge on my part.

**Review by penny-lischer — APPROVED** [review]
lgtm

**Review by yudonoodles — APPROVED** [review]
Looks good - thanks Etan! 

Only requested change is the H2 uses primary-dark #1A4480. This will be the same case for both case studies and News headings moving forward. This slight styling to the headings helps them stick out as articles more and differentiate from the usual H2 styling on other pages.

**Review by audreykwr — APPROVED** [review]

**Pull request merged**